src/common/libflux/test/future.c

```diff
@@ -10,7 +10,7 @@
 
 int aux_destroy_called;
 void *aux_destroy_arg;
-void aux_destroy (void *arg)
+void aux_destroy_fun (void *arg)
```

Comment:
i don't know how to comment on a commit message, but noticed you have at typo ```wiht``` in the commit message for this change :-)